### PR TITLE
Clean up code results

### PR DESCRIPTION
**Pull request by gyutaepark**
To be reviewed after #10.

- Changes output sqlite file name to 'coordinates' from 'test'
- Changes output sqlite table to 'reactor_coordinates' from test
- Removes index column

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
 
 This is under a CC-BY license. 
+
+
+## How-to-use
+
+Run:
+`python scraping_wikidata.py`
+
+Output:
```
Comment (jbae11) [inline README.md]: Output File:

**Review by jbae11 — APPROVED** [review]
Looks good

File: README.md (hunk 1 above)
Comment (gyutaepark) [reply README.md]: Thank you!

**Comment by gyutaepark** [comment]
Merging now.

**Pull request merged**